libs/cosmos-sdk/store/iavl/store_test.go

```diff
@@ -581,18 +581,11 @@ func testCommitDelta(t *testing.T) {
 	assert.EqualValues(t, 2, cid1.Version)
 	assert.NotEqual(t, emptyDelta, treeDelta1)
 
-	// use delta and produce delta
```

Comment:
Isn't the hash and version of the new height a definite value when there is no data update?I'm not sure whether this was a bug of delta